lib/index.js

```diff
@@ -77,6 +77,22 @@ class Client {
     return ActionApi.action(this.api, this.apiKey, actionType, data)
   }
 
+  /**
+   * Retrieves the dataset
+   * @param {string} nameOrId - Id or name of the dataset
+   */
+  async retrieve(nameOrId) {
+    const response = await this.action(
+      'package_show',
+      {
+        nameOrId,
```

Comment:
Trailing comma ...

BTW: i would have kept those variables you had but made them internal to the method so that people reading the code knew that was happening but it is not a big deal and we can leave as is.

Reply:
For trailing commas I followed https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma. Also this is configured in prettierr config